### ISSUE TITLE
Auto-save device when it is loaded from a non-standard location (Trac #126)

### ISSUE THREAD

**Issue by cfobel** [wheeler-microfluidics/microdrop#130]
This will ensure that when the program is closed and restarted, the device will exist in the "Microdrop/devices" directory.

Migrated from http://microfluidics.utoronto.ca/microdrop/ticket/126
```json
{
    "status": "closed", 
    "changetime": "2014-04-17T19:39:43", 
    "description": "This will ensure that when the program is closed and restarted, the device will exist in the \"Microdrop/devices\" directory.", 
    "reporter": "ryan", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1397763583719301", 
    "component": "microdrop", 
    "summary": "Auto-save device when it is loaded from a non-standard location", 
    "priority": "minor", 
    "keywords": "", 
    "version": "0.1", 
    "time": "2012-10-18T18:34:13", 
    "milestone": "Microdrop 1.0", 
    "owner": "", 
    "type": "defect"
}
```

**Comment by cfobel** [comment]
Trac update at 20121022T18:15:20: 
* ryan commented:

> This is fixed as of commit b7534786ad12b8cb5db83f9342bafbf32df7f6c2 (device is marked as modified and user prompted to save on exit).


* ryan changed resolution from "" to "fixed"
* ryan changed status from "new" to "closed"

**Comment by cfobel** [comment]
Trac update at 20140417T19:39:43: ryan changed component from "" to "microdrop"
